### DISCUSSION TITLE
Team Plan - question

### DISCUSSION THREAD

**Discussion by gustavosantaella** [community/community#39843] [Other Feature Feedback, Questions, & Ideas]
I have a problem with the "Team Plan". I did the pay of it.

I'm trying to create a new organization but I can't, why ?

**Comment by ossanna16** [comment]
Hi there @gustavosantaella and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
